src/test/java/hw2/NumberHolderTest.java

```diff
@@ -0,0 +1,29 @@
+package hw2;
+
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class NumberHolderTest {
+
+    NumberHolder numberHolder = new NumberHolder(7, 3.58F);
```

Comment:
Please convert these to the local variables (move each one to the test method and name just numberHolder) - we do not use them between all methods